cmd/fluxd/main.go

```diff
@@ -508,7 +513,11 @@ func main() {
 			logger.Log("err", err)
 			os.Exit(1)
 		}
-		k8sManifests = kubernetes.NewManifests(namespacer, logger)
+		if *sopsEnabled {
+			k8sManifests = kubernetes.NewManifests(namespacer, logger)
+		} else {
+			k8sManifests = kubernetes.NewSopsManifests(namespacer, logger)
```

Comment:
Shouldn't this be the other way around? (Sorry I missed this)

Reply:
Whoops, my bad. You didn't miss it. I just passed `*sopsEnabled` to the `NewSopsManifests` call before.

Good catch

Reply:
Fixed now